File: Aluguel de carro.cpp
#include <iostream>
#include <iomanip>
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int aluguel = 30.00;
int kmaluguel = 0.01;
double valoratual;
double desconto;
double diasutilizado;
double kmsutilizados;

using namespace std;


int main() {

cin>>diasutilizado;
cin>>kmsutilizados;

valoratual = (diasutilizado * aluguel) + (kmaluguel * kmsutilizados);
desconto = valoratual * (0.1);

cout<<fixed<<setprecision(2)<<valoratual-desconto<<endl;

}

File: Alugueldecarro.py
aluguel = float (30.00)
kmaluguel = float (0.01)
valoratual = float
desconto = float


diasutilizado = float (input())
kmsutilizados = float (input())


valoratual = (diasutilizado * aluguel) + (kmaluguel * kmsutilizados)
desconto = valoratual * (0.1)

print("%.2f"%(valoratual-desconto))

#Aluguel de carro
